app/views/refinery/image_slideshows/admin/image_slides/_image_slide.html.erb

```diff
@@ -23,6 +24,10 @@
     <% end %>
   </span>
 
+  <%= content_tag :span, class: "#{image_slide.active? ? :success_icon : :failure_icon} less-important" do %>
```

Comment:
We have label to handle `draft` display: https://github.com/refinery/refinerycms/blob/88f3eee61914d548dce5e7c1d1f8d3a57a0c9ece/pages/app/helpers/refinery/admin/pages_helper.rb#L51-L53